actions/npm_publish_version.meta.yaml

```diff
@@ -0,0 +1,25 @@
+---
+  name: "npm_publish_version"
+  runner_type: "action-chain"
+  description: "Published new versions to npm."
+  enabled: true
+  entry_point: "workflows/npm_publish_version.yaml"
+  parameters:
+    repo:
+      type: "string"
+      description: "Url of the repo to clone"
+      default: "git@github.com:StackStorm/hubot-stackstorm.git"
+      required: true
+    branch:
```

Comment:
This will only operate on a release version branch like elsewhere, right?

(Just want to confirm, since operating on non-release branch wouldn't work well without also specifying a commit sha)